terminal-colors/material-palenight.terminal

```diff
@@ -0,0 +1,200 @@
+    dmVy0RcYVHJvb3SAAQgRGiMtMjc7QUhOW2KMjpCVoKmxtL3P0tcAAAAAAAABAQAAAAAA
+    AAAZAAAAAAAAAAAAAAAAAAAA2Q==
+    </data>
+    <key>ANSIYellowColor</key>
+    <data>
+    YnBsaXN0MDDUAQIDBAUGFRZYJHZlcnNpb25YJG9iamVjdHNZJGFyY2hpdmVyVCR0b3AS
+    AAGGoKMHCA9VJG51bGzTCQoLDA0OVU5TUkdCXE5TQ29sb3JTcGFjZVYkY2xhc3NPECcx
+    LjAwMDAwMDAwMDAgMC43OTYwNzg0MzE0IDAuNDE5NjA3ODQzMQAQAYAC0hAREhNaJGNs
+    YXNzbmFtZVgkY2xhc3Nlc1dOU0NvbG9yohIUWE5TT2JqZWN0XxAPTlNLZXllZEFyY2hp
+    dmVy0RcYVHJvb3SAAQgRGiMtMjc7QUhOW2KMjpCVoKmxtL3P0tcAAAAAAAABAQAAAAAA
+    AAAZAAAAAAAAAAAAAAAAAAAA2Q==
+    </data>
+    <key>BackgroundColor</key>
+    <data>
+    YnBsaXN0MDDUAQIDBAUGFRZYJHZlcnNpb25YJG9iamVjdHNZJGFyY2hpdmVyVCR0b3AS
+    AAGGoKMHCA9VJG51bGzTCQoLDA0OVU5TUkdCXE5TQ29sb3JTcGFjZVYkY2xhc3NPECcw
+    LjEyOTQxMTc2NDcgMC4xMjk0MTE3NjQ3IDAuMTI5NDExNzY0NwAQAYAC0hAREhNaJGNs
+    YXNzbmFtZVgkY2xhc3Nlc1dOU0NvbG9yohIUWE5TT2JqZWN0XxAPTlNLZXllZEFyY2hp
+    dmVy0RcYVHJvb3SAAQgRGiMtMjc7QUhOW2KMjpCVoKmxtL3P0tcAAAAAAAABAQAAAAAA
+    AAAZAAAAAAAAAAAAAAAAAAAA2Q==
+    </data>
+    <key>CursorColor</key>
+    <data>
+    YnBsaXN0MDDUAQIDBAUGFRZYJHZlcnNpb25YJG9iamVjdHNZJGFyY2hpdmVyVCR0b3AS
+    AAGGoKMHCA9VJG51bGzTCQoLDA0OVU5TUkdCXE5TQ29sb3JTcGFjZVYkY2xhc3NPECcw
+    LjkyNTQ5MDE5NjEgMC45MzcyNTQ5MDIwIDAuOTQ1MDk4MDM5MgAQAYAC0hAREhNaJGNs
+    YXNzbmFtZVgkY2xhc3Nlc1dOU0NvbG9yohIUWE5TT2JqZWN0XxAPTlNLZXllZEFyY2hp
+    dmVy0RcYVHJvb3SAAQgRGiMtMjc7QUhOW2KMjpCVoKmxtL3P0tcAAAAAAAABAQAAAAAA
+    AAAZAAAAAAAAAAAAAAAAAAAA2Q==
+    </data>
+    <key>SelectionColor</key>
+    <data>
+    YnBsaXN0MDDUAQIDBAUGFRZYJHZlcnNpb25YJG9iamVjdHNZJGFyY2hpdmVyVCR0b3AS
+    AAGGoKMHCA9VJG51bGzTCQoLDA0OVU5TUkdCXE5TQ29sb3JTcGFjZVYkY2xhc3NPECcw
+    LjUzNzI1NDkwMjAgMC44NjY2NjY2NjY3IDEuMDAwMDAwMDAwMAAQAYAC0hAREhNaJGNs
+    YXNzbmFtZVgkY2xhc3Nlc1dOU0NvbG9yohIUWE5TT2JqZWN0XxAPTlNLZXllZEFyY2hp
+    dmVy0RcYVHJvb3SAAQgRGiMtMjc7QUhOW2KMjpCVoKmxtL3P0tcAAAAAAAABAQAAAAAA
+    AAAZAAAAAAAAAAAAAAAAAAAA2Q==
+    </data>
+    <key>TextBoldColor</key>
+    <data>
+    YnBsaXN0MDDUAQIDBAUGFRZYJHZlcnNpb25YJG9iamVjdHNZJGFyY2hpdmVyVCR0b3AS
+    AAGGoKMHCA9VJG51bGzTCQoLDA0OVU5TUkdCXE5TQ29sb3JTcGFjZVYkY2xhc3NPECcw
+    LjkyNTQ5MDE5NjEgMC45MzcyNTQ5MDIwIDAuOTQ1MDk4MDM5MgAQAYAC0hAREhNaJGNs
+    YXNzbmFtZVgkY2xhc3Nlc1dOU0NvbG9yohIUWE5TT2JqZWN0XxAPTlNLZXllZEFyY2hp
+    dmVy0RcYVHJvb3SAAQgRGiMtMjc7QUhOW2KMjpCVoKmxtL3P0tcAAAAAAAABAQAAAAAA
+    AAAZAAAAAAAAAAAAAAAAAAAA2Q==
+    </data>
+    <key>TextColor</key>
+    <data>
+    YnBsaXN0MDDUAQIDBAUGFRZYJHZlcnNpb25YJG9iamVjdHNZJGFyY2hpdmVyVCR0b3AS
+    AAGGoKMHCA9VJG51bGzTCQoLDA0OVU5TUkdCXE5TQ29sb3JTcGFjZVYkY2xhc3NPECcw
+    LjkyNTQ5MDE5NjEgMC45MzcyNTQ5MDIwIDAuOTQ1MDk4MDM5MgAQAYAC0hAREhNaJGNs
+    YXNzbmFtZVgkY2xhc3Nlc1dOU0NvbG9yohIUWE5TT2JqZWN0XxAPTlNLZXllZEFyY2hp
+    dmVy0RcYVHJvb3SAAQgRGiMtMjc7QUhOW2KMjpCVoKmxtL3P0tcAAAAAAAABAQAAAAAA
+    AAAZAAAAAAAAAAAAAAAAAAAA2Q==
+    </data>
+    <key>name</key>
+    <string>terminal.sexy</string>
```

Comment:
Out of curiosity, what is this value? Was this copied over? Looks like it's in all the .terminal files

Reply:
Hey, I have been using: https://terminal.sexy/ which internally uses https://github.com/stayradiated/termcolors.
It would probably be very useful to write a script that does the conversion automatically.